src/styles/blocks/page.scss

```diff
@@ -1,3 +1,5 @@
+$trans-dur: 300ms;
```

Comment:
It's better to use full words for variable names: `$transition-duration`. It will be more readable for other teammates in your team.